OneSignalSDK/onesignal/src/main/java/com/onesignal/LocationGMS.java

```diff
@@ -189,7 +194,7 @@ static void getLocation(Context context, boolean promptLocation, LocationHandler
 
```

Comment:
```
String alertDialogTitle = context.getString(R.string.location_not_available_title);
String alertDialogMessage = context.getString(R.string. location_not_available_message);
```
Pass these into both usages of
`sendAndClearPromptHandlers(promptLocation, false, alertDialogTitle, alertDialogMessage);`